containerize/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG BUILDIMAGE
+FROM ${BUILDIMAGE} as builder
+
+ARG BUILD_DATE
+ARG ANALYSIS_NAME=analysis
+LABEL   org.label-schema.build-date=$BUILD_DATE \
+        org.label-schema.name="$ANALYSIS_NAME Docker image" \
+        org.label-schema.description="Provides a runnable CMSSW images with $ANALYSIS_NAME pre-installed." \
+        org.label-schema.vendor="FNAL"
+
+USER    cmsusr
+WORKDIR /home/cmsusr
+
+ARG CMSSW_VERSION
+ARG TAR
+
+COPY ${TAR} /home/cmsusr/${CMSSW_VERSION}.tar.gz
+RUN pwd && \
+	ls -alh ./ && \
+	source /cvmfs/cms.cern.ch/cmsset_default.sh && \
+	echo "Unpacking ${CMSSW_VERSION}.tar.gz into ${PWD} ..." && \
+	tar -xzf ${CMSSW_VERSION}.tar.gz && \
+	ls -alh ./ && \
+	rm ${CMSSW_VERSION}.tar.gz && \
+	ls -alh ./ && \
```

Comment:
I removed one of the `ls`, but I like having the other two because it's useful for debugging purposes. Plus, it really doesn't add too much to the logs.